src/feature/hs/hs_config.c

```diff
@@ -334,6 +348,48 @@ config_service_v3(const config_line_t *line_,
       export_circuit_id = true;
       continue;
     }
+    if (!strcasecmp(line->key, "HiddenServiceEnableIntroDoSDefense")) {
```

Comment:
Perhaps we need some unittests for the torrc parsing and validation? And maybe cross-checking them with the intro-point validation checks?

Reply:
See commit 667e6db249